apps/src/blockly/components/ModalFunctionEditor.jsx

```diff
@@ -28,22 +29,27 @@ export default function ModalFunctionEditor() {
             type="button"
             id={MODAL_EDITOR_DELETE_ID}
             onClick={emptyOnClick}
-            color={Button.ButtonColor.brandSecondaryDefault}
+            color={Button.ButtonColor.white}
+            style={buttonStyles}
             size={buttonSize}
-          >
-            {msg.delete()}
```

Comment:
The reason the text wasn't being centered is because we were adding it directly to the button div. It is meant to be supplied to the button via the `text` property and added to a span contained within the div.

Before (mid-task):
![image (167)](https://github.com/code-dot-org/code-dot-org/assets/43474485/8d4a433a-ae3c-46ba-87c9-e138b7e5e9a0)

After change:
![image](https://github.com/code-dot-org/code-dot-org/assets/43474485/2e725e39-7244-47b3-a434-1bd65fdfcf72)